File: sdk-dotnet/Examples/BasicExample/MyWorker.cs
using LittleHorse.Sdk.Worker;

namespace Examples.BasicExample
{
    public class MyWorker
    {
        [LHTaskMethod("greet")]
        public async Task<string> Greeting(string name)
        {
            var message = $"Hello team, This is a Dotnet Worker";
            Console.WriteLine($"Executing task greet {name}: " + message);
            return message;
        }
    }
}

File: examples/java/run-child-workflow/src/main/java/io/littlehorse/examples/MyWorker.java
package io.littlehorse.examples;

import io.littlehorse.sdk.worker.LHTaskMethod;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class MyWorker {

    private static final Logger log = LoggerFactory.getLogger(MyWorker.class);

    @LHTaskMethod("greet")
    public String greeting(String name) {
        System.out.println("Executing task greet: " + name);
        return "hello there, " + name;
    }

}
